vita3k/gui/src/game_install_dialog.cpp

```diff
@@ -93,9 +94,14 @@ void draw_game_install_dialog(GuiState &gui, HostState &host) {
         ImGui::PopStyleColor();
         ImGui::TextColored(GUI_COLOR_TEXT, "Succesfully installed game.\n%s [%s] %s", host.io.title_id.c_str(), host.game_title.c_str(), host.game_version.c_str());
         ImGui::Separator();
+        ImGui::Checkbox("Delete Game file installer?", &delete_game_file);
```

Comment:
Hmm... I'm not sure what you mean here...

Did you mean `Delete vpk after installation?`